### ISSUE TITLE
new rule: react-a11y-access-keys

### ISSUE THREAD

**Issue by HamletDRC** [microsoft/tslint-microsoft-contrib#200]
Accesskey attribute values should be unique
    • http://oaa-accessibility.org/wcag20/rule/1/

Accesskey attribute values should not interfere with IE shortcuts
    • http://oaa-accessibility.org/wcag20/rule/2/

Enforce that the accessKey prop is not used on any element to avoid complications with keyboard commands used by a screenreader. From  no-access-key.md
    • https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-access-key.md

Partially implemented in no-access-key.md - https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-access-key.md

**Comment by JoshuaKGoldberg** [comment]
### 💀 _**It's time!**_ 💀

**TSLint is being deprecated**; therefore, it and tslint-microsoft-contrib are no longer accepting pull requests for major new changes or features. See #4534. 😱

If you'd like to see this change implemented, you have two choices:
* **Recommended:** Check if this is available in [ESLint](https://eslint.org) + [typescript-eslint](https://typescript-eslint.io) ✅
* _Not Recommended: Fork TSLint or tslint-microsoft-contrib locally_ 🤷‍♂️

👋 It was a pleasure open sourcing with you!

_If you believe this message was posted here in error, please comment so we can re-open the issue!_
